lint/style.py

```diff
@@ -87,16 +73,14 @@ def wrapper(*args):
             if key != "icon":
                 return res
             else:
-                # returning paths
                 if res in ("circle", "dot", "bookmark", "none"):  # Sublime Text has some default icons
                     return res
-                elif res != os.path.basename(res):
-                    return res
                 else:
-                    icon_path = GUTTER_ICONS["icons"].get(res)
-                    if icon_path:
-                        return icon_path
-                return GUTTER_ICONS["icons"][error_type]
+                    theme = persist.settings.get('gutter_theme')
```

Comment:
I thought (in general keeping this PR) but after here you just 

```
	return 'Packages\SublimeLinter\gutter-themes\{}\{}.png'.format(theme, res)
```

Reply:
Oh, that's also good. If the specified icon doesn't exist it will fail like this:

<img width="104" alt="screen shot 2018-02-28 at 14 23 16" src="https://user-images.githubusercontent.com/2543659/36789785-043bbe2c-1c93-11e8-913e-7f598449ad16.png">

```
Unable to decode Packages/SublimeLinter/gutter-themes/Hands/pointer-outline.png
```

Which actually is a really nice way to fail IMO.

Reply:
👍